src/main/steps/common/dashboard-page.njk

```diff
@@ -25,7 +25,7 @@
 
 
 {% block pageTitle %}
-  {{ serviceName }} - {{ title }} - {{ govUk }}
+  {{ headingTitle }}{{ title }} - Private law - GOV.UK
```

Comment:
Do we need both headingTitle and title on the dashboard page. if so, there is a hyphen missing between headingTitle & title.

Reply:
added necessary changes